OpenImis.ePayment/Escape/Payment/Controllers/PaymentController.cs

```diff
@@ -117,6 +117,29 @@ public IActionResult GetReconciliation([FromBody] gepgSpReconcResp model)
                 var gepgFile = new GepgFoldersCreating("Reconc", "Data", reconc, env);
                 gepgFile.putToTargetFolderPayment();
 
+                foreach (var recon in model.ReconcTrxInf) 
+                {
+                    var paymentToCompare = imisPayment.GetPaymentToReconciliate(new { billId = recon.SpBillId, paidAmount = recon.PaidAmt });
+                    if (paymentToCompare != null)
+                    {
+                        int billId = (int) paymentToCompare.GetType().GetProperty("paymentStatus").GetValue(paymentToCompare);
+                        if (billId < 5)
```

Comment:
I think this is a comparation between a PaymentId and a PaymentStatus. Is this correct?

Reply:
Yes. You are right. My gap. I fix this!